docs/docs/PluginDevelopment.md

```diff
@@ -21,24 +20,31 @@ For example:
 ```bash
     dvp build
 ```
-    
-   4. Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:
+
+
+Upload plugin
+----------------
+   Upload the `artifacts.json` ( generated in step 3 ) on Delphix Engine:

```

Comment:
nit: I think the artifact is called `artifact.json`?

Reply:
Fixed